stdlib.sh

```diff
@@ -1567,16 +2191,31 @@ function __STDLIB_API_1_std::capture() { # {{{
 			;;
 	esac
 
-	if ! type -t "${cmd:-}" >/dev/null; then
-		error "Invalid parameters: <command> '${cmd:-}' not found"
-		std_ERRNO=$( errsymbol EARGS )
-		return 1
+	## NB: 'type' check requires that args are passed separately from the cmd
+	##     to be executed. This may render specifying subcommands complex,
+	##     especially when one wants to prepend them with env variables like
+	##     DEBUG=1. Therefore, we can either pass cmd and args separately and
+	##     use the old logic to check availability of the command being invoked,
+	##     or pass the subcommand as a single param and skip the check.
+	if [[ ! -z "${args}" ]]; then
+		if ! type -t "${cmd:-}" >/dev/null; then
+			error "Invalid parameters: <command> '${cmd:-}' not found"
+			std_ERRNO=$( errsymbol EARGS )
+			return 1
+		fi
 	fi
 
 	type -pf stdbuf >/dev/null 2>&1 && stdbuf="stdbuf -eL"
 
-	response="$( eval "${stdbuf:+${stdbuf} }${cmd} ${args[*]} ${redirect}" )" ; rc=${?}
-	output "${response:-}"
+	## Subshell execution with output capture prevents interactive mode
```

Comment:
For when you need to provide input to the subshell, like typing password or answering to "y/n" questions. Any better alternative is welcomed!